import-dashboards.py

```diff
@@ -105,7 +105,7 @@ def add_datasources():
             sys.exit(-1)
 
     if 'CloudWatch' not in ds:
-        data = json.dumps({'name': 'CloudWatch', 'type': 'cloudwatch', 'jsonData': '{"defaultRegion":"us-east-1"}', 'access': 'proxy', 'isDefault': False})
+        data = '{"name":"CloudWatch","type":"cloudwatch","url":"","access":"proxy","jsonData":{"timeField":"@timestamp","authType":"keys","defaultRegion":"us-east-1"},"secureJsonFields":{}}'
```

Comment:
Any particular reason why you don't use `dict` and `json.dumps()`?

Reply:
it is fast proof-of-concept, feel free to rework code as you wish for your ticket ;)